ui/perfherder/graphs/TableView.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import ReactTable from 'react-table';
+import moment from 'moment';
+import { groupBy, forIn } from 'lodash';
+
+const TableView = ({ testData, highlightAlerts, highlightedRevisions }) => {
+  const allDataPoints = [];
+  const tableDataPoints = [];
+
+  const columns = [
+    {
+      Header: 'Date',
+      accessor: 'date',
+      headerClassName: 'table-header',
+    },
+  ];
+
+  testData.forEach((item, index) => {
+    if (item.visible) {
+      // unique key object name
+      const id = `name${index}`;
+      const value = `value${index}`;
+
+      // add columns for each test
+      columns.push({
+        Header: (
+          <span>
+            <p className="font-weight-bold mb-1">{item.name}</p> <br />
+            {item.platform} | {item.repository_name}{' '}
+          </span>
+        ),
+        Cell: ({ original }) => {
+          return <div>{original[value]}</div>;
+        },
+        headerClassName: `table-header ${item.color[0]}`,
+        getProps: (state, rowInfo) => ({
+          style: {
+            background: rowInfo && rowInfo.original.highlighted && '#ffde2f',
+          },
+          'aria-label': 'highlighted revision',
```

Comment:
A thought I had... if someone enters a revision, should the table results filter to that revision(s) or otherwise jump to it first - since that is what someone would want to know about?

Reply:
That's a good idea. I think the second suggestion would be the best one. I took note of that and I will try to implement it.